llvm/test/CodeGen/DirectX/rootsignature-validation-binding-limits-upperbound.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: opt -S -passes='dxil-post-optimization-validation' -mtriple=dxil-pc-shadermodel6.6-compute %s 2>&1 | FileCheck %s
+; This is a valid code, it checks the limits of a binding space
+; CHECK-NOT: error: .*
```

Comment:
This doesn't use a regex, so it would only match if we output the exact string "error: .*".